### ISSUE TITLE
repmat depracated in visualizeCellVectors?

### ISSUE THREAD

**Issue by S-AM-I** [simulkade/JFVMvis.jl#1]
I think repmat is deprecated in this current version of Julia, so I have edited the `visualizeCellVectors` function for a domain of 2 or 2.5 to:

```Julia
x = phi.domain.cellcenters.x
y = phi.domain.cellcenters.y
quiver(repeat(x, 1, length(y)), repeat(y', length(x), 1), phi.xvalue[1:end-1,1:end], phi.yvalue[1:end,1:end-1])
```
Where repmat has been replaced with repeat, which seems to work for me

**Comment by simulkade** [comment]
Thanks! May I ask you to create a pull request with these suggested changes?

**Comment by simulkade** [comment]
Fixed with #2 by @SA8416
